typednavigation/src/main/java/com/xmartlabs/typednavigation/TypedNavigationInterface.kt

```diff
@@ -3,54 +3,137 @@ package com.xmartlabs.typednavigation
 import androidx.navigation.NamedNavArgument
 import androidx.navigation.NavType
 
-
+/** An interface that implements the basic attributes needed for navigation
```

Comment:
don't you need a nl?
```suggestion
/**
An interface that implements the basic attributes needed for navigation
```